### ISSUE TITLE
Create independent repo for vSphere charts

### ISSUE THREAD

**Issue by PennyScissors** [rancher/rancher#35702]
Create an independent rancher/vsphere-charts repo for the CPI and CSI vsphere charts. This repo should work similarly to how [harvester/charts ](https://github.com/harvester/charts) works.

**Comment by deniseschannon** [comment]
@samkulkarni20 After your team is unblocked with vsphere, can QA validate this repo could be added into Rancher as a new git repo and deployed onto a RKE1 cluster?

**Comment by samkulkarni20** [comment]
@deniseschannon We won't be adding this new repo directly to rancher Apps & Marketplace configuration. This will only serve as an upstream vsphere CPI/CSI repo for `rancher/charts`. Previously those charts were "local" for `rancher/charts`, istead of that it'll change to an upstream url pointing to `rancher/vsphere-charts` commit hash.
Hence the testing flow will remain unchanged.

**Comment by deniseschannon** [comment]
@samkulkarni20 We should and can test the charts before the rancher/charts is pointed to this new version. We should test that these new versions of the charts are working as expected as RKE2 and Rancher will be picking this up. 

There is a separate ticket tracking picking up and using these charts in the actual rancher/charts, but that shouldn't have prevented us from testing the independent charts before this change was made.

**Comment by samkulkarni20** [comment]
Oh, I wasn't aware we do not plan to integrate the new repo with `rancher/charts` just yet. I thought, this ticket is to do both. I'll ask QA to start validating the new repo on its own.
@deniseschannon

**Comment by deniseschannon** [comment]
@samkulkarni20 We plan to do both. The independent repo will be consumed by the RKE2 team AND integrated with rancher/charts.

**Comment by vivek-infracloud** [comment]
Logged an [issue](https://github.com/rancher/dashboard/issues/5269#issue-1156959813) observed during testing.

**Comment by PennyScissors** [comment]
There is no need to test this issue separately. This will be indirectly validated when issues https://github.com/rancher/rancher/issues/33848, https://github.com/rancher/rancher/issues/36081, and https://github.com/rancher/rancher/issues/36081 are tested because `rancher/charts` is already pointing to the new CPI and CSI in the `rancher/vsphere-charts` repository. If the new charts charts work, then the independent repository works.
cc @sowmyav27

**Comment by doflamingo721** [comment]
I have added the testing instructions for this issue [here](https://github.com/rancher/rancher/issues/33848#issuecomment-1061646342) on https://github.com/rancher/rancher/issues/33848
cc: @vivek-infracloud

**Comment by deniseschannon** [comment]
Repo now exists and is being updated by the team to get picked up by Rancher and RKE2 teams independently. 

https://github.com/rancher/vsphere-charts
